spec/services/charges/charge_models/volume_service_spec.rb

```diff
@@ -33,48 +33,96 @@
   context 'when aggregation is 0' do
     let(:aggregation) { 0 }
 
-    it 'does not apply the flat amount' do
+    it 'does not apply the flat amount', :aggregate_failures do
       expect(apply_volume_service.amount).to eq(0)
+      expect(apply_volume_service.unit_amount).to eq(0)
+      expect(apply_volume_service.amount_details).to eq(
+        {
+          flat_amount: 0,
+          per_unit_amount: 0,
+          per_unit_total_amount: 0,
+        }
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)